Packs/FortiSandbox/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### FortiSandbox
+- Fixed an issue where ***fortisandbox-upload-file*** command were failing due to file not found error.
+- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
```

Comment:
If we changed the currentVersion, let's also change the name of this RN to 1_0_3.md

Reply:
Done